sympy/assumptions/handlers/order.py

```diff
@@ -287,23 +288,22 @@ def Add(expr, assumptions):
     def Pow(expr, assumptions):
         if expr.is_number:
             return AskPositiveHandler._number(expr, assumptions)
+        if expr.base is S.Exp1:
+            if ask(Q.real(expr.exp), assumptions):
+                return True
+            if ask(Q.imaginary(expr.exp), assumptions):
```

Comment:
Note that `exp(2*pi*I) == 1`.

Reply:
That's true, I have just copied the code from the method `exp`. It probably need to check whether `expr.exp` is a multiple of `2*pi*I`, right?

Reply:
Actually we should test if the imaginary part is a multiple of `2*pi*I`. But would that take too much time?

Reply:
This issue should be separated from this PR.